domains/client/relayer/src/worker.rs

```diff
@@ -193,23 +193,23 @@ where
             // mark this block processed and continue to next one.
             if !can_relay_message_from_block(chain_id, number)? {
                 Relayer::store_relayed_block(&client, chain_id, number, hash)?;
-                tracing::info!(
+                tracing::debug!(
                     target: LOG_TARGET,
                     "Chain({chain_id:?}) messages in the Block ({number:?}, {hash:?}) cannot be relayed. Skipping...",
                 );
             } else {
                 match message_processor(&client, hash) {
                     Ok(_) => {
                         Relayer::store_relayed_block(&client, chain_id, number, hash)?;
-                        tracing::info!(
+                        tracing::debug!(
```

Comment:
I think this would be beneficial to keep it `info` else there is no feedback from relayer if its processing blocks. Though not a strong opinion from my end

Reply:
This log will print for every domain block and may be annoying even for the relayer who don't always want to know this since there is nothing it can do if the log show it is lagging/slowing.

Or perhaps we can provide a domain subcommand to query the relaying progress, so the relayer can check manually the progress if they really want to know.

Reply:
its fine for now. Not a blocker for me